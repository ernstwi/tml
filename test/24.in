init foo

default edge
    guard { x == 10 }
    guard { x == 10 }
    sync { a }
    sync { a }
    sync { a }
    reset { x }
    reset { x }
    reset { x }
    reset { x }
