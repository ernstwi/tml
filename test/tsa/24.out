Semantic error: [default edge]: 2 guards
Semantic error: [default edge]: 3 syncs
Semantic error: [default edge]: 4 resets
